docs/connecting.txt

```diff
@@ -51,11 +51,32 @@ the following two formats are both allowed::
     jdbc:crate://localhost:5432/
 
 
-In order to connect to multiple servers multiple ``<host>:<psql-port>``
-pairs can be specified by delimit them using a comma::
+To support connection fail-over, multiple pairs of <host>:<psql-port> dilimited
+by a comma can be specified::
 
     crate://host1.example.com:5432,host2.example.com:5432/
 
+The driver will try to connect to the hosts in the order defined by the comma
+separated list until the first successfull connection is established.
+
+.. note::
+
+    If you have multiple client apps/threads connecting to the CrateDB cluster,
+    it is adviseable to use shuffle the order of those pairs in each client 
+    app/thread, which will allow a convenient connection load balancing accross
+    the cluster, instead of having all clients connecting to the same node.
+    An easy way to achieve this is to randomly shuffle the list in each client: 
```

Comment:
drop "An easy way..." onwards. repetition of previous point

instead, add a new para and put

"Here's one way to shuffle the list of hosts in Java:"